src/base/tz_support.c

```diff
@@ -5469,3 +5469,25 @@ conv_tz (void *p_out, const void *p_in, DB_TYPE type)
 
   return err_status;
 }
+
+//TODO: make tz_get_offset_in_mins get into account DST
+/*
+ * tz_get_offset_in_mins () - time zone offset in minutes from GMT
+ */
+int
+tz_get_offset_in_mins ()
+{
+  time_t currtime;
+  struct tm *timeinfo;
+
+  time (&currtime);
+  timeinfo = gmtime (&currtime);
+  time_t utc = mktime (timeinfo);
+  timeinfo = localtime (&currtime);
+  time_t local = mktime (timeinfo);
+
+  // Get offset in minutes from UTC
+  double offsetFromUTC = difftime (utc, local) / 60;
```

Comment:
I am a little confused. But isn't it correct to call difftime (local, utc)?

Reply:
I think timezones are defined as (gmt - local)